### ISSUE TITLE
Changesets cannot be "duplicated" because UUIDs are, of course, unique

### ISSUE THREAD

**Issue by postphotos** [xwp/wp-customize-snapshots#165]
If using a plugin like "Duplicate Posts" and you clone a Changeset and then try to edit it, it throws an error: `Invalid changeset UUID`. 

1) This is perfectly valid behavior and it makes sense why, but can we, in the meantime, provide a warning in the release notes? 
2) Further, can we add functionality to prevent plugins like "Duplicate Posts" from using this post type in the method followed in time for the `4.9 WP Core` release?
3) Finally, can we add a working `duplicate` feature? **Use Case: Imagine having `Changeset-1` getting cloned and pruned into `Changeset-1a` and `Changeset-1b`.**
4) Similar functionality to merge `Changeset-2` + `Changeset-3` into `Changeset[2 +3]` would be great as well.

**Comment by westonruter** [comment]
Actually, providing a way to copy a changeset has already been suggested in #41 with implementation in pending PR #90.

**Comment by postphotos** [comment]
Yes! Thanks @PatelUtkarsh @westonruter for your awesome contributions! - this is also related to #118 and #93 as well. Essentially, we're after a similar branching pattern like we see in git or other structured version control systems. Split, merge, fork, branch - as much as we can, mirror workflows that folks are actually using in publishing and dev spaces.
